packages/example/src/utils/dataSeries.generator.tsx

```diff
@@ -73,4 +73,27 @@ export class DataSeriesGenerator {
         .sort((a, b) => a.x - b.x),
     }));
   }
+
+  generateYY(): DataSeries<XYPoint<[number, number]>>[] {
```

Comment:
shouldn't it be `XY`? 🤔

Reply:
nope, it's intentional, wanted it to signify that there would be two `y` values generated, maybe I can rename it to `generateXYY` and the base one to `generateXY`, wdyt?